### PR TITLE
Update README for OSS usage

### PR DESCRIPTION
**Pull request by bidetofevil**
## Goal

Update project README for users who are not Embrace customers. Given that folks who discover us via our repo will likely want to try the SDK without signing up for Embrace first, I've made the README more geared towards getting them going.

This should help folks coming to the repo for the first time after hearing about us just to fork and play with the SDK.

### CODE REVIEW

**Comment by bidetofevil** [comment]
* **#983** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/983?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bidetofevil and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: README.md [hunk 1]
```diff
@@ -8,34 +8,85 @@
...
+
+1. In your app's Gradle file, add a dependency to the latest version of the Embrace Swazzler Gradle plugin: `io.embrace:embrace-swazzler:<version>`.
+   - This plugin is responsible for configuring your app at build time to auto-capture telemetry. This includes:
+     - Updating dependencies to include optional modules that are needed for certain features.
+     - Setting up configuration files to be read at runtime.
+     - Doing bytecode instrumentation to enable the capture of certain telemetry.
+2. In the Gradle files of modules you want to invoke Embrace SDK API methods, add a dependency to the main Embrace SDK module: `'io.embrace:embrace-android-sdk:<version>`.
+3. In the `main` directory of your app's root source folder (i.e. `app/src/main/`), add in a file called `embrace-config.json` that contains `{}` as its only line.
+   - To further configure the SDK, additional attributes can be added to this configuration file. 
+   - See [this page](https://embrace.io/docs/android/features/configuration-file/) for further details.
```
Comment (davidlawrencer) [inline README.md:37]: See [documentation]

File: README.md [hunk 2]
```diff
@@ -8,34 +8,85 @@
...
 [![android api](https://img.shields.io/badge/Android_API-21-green.svg "Android min API 21")](https://dash.embrace.io/signup/)
 [![build](https://img.shields.io/github/actions/workflow/status/embrace-io/embrace-android-sdk/ci-gradle.yml)](https://github.com/embrace-io/embrace-android-sdk/actions)
+[![latest version](https://shields.io/github/v/release/embrace-io/embrace-android-sdk)](https://shields.io/github/v/release/embrace-io/embrace-android-sdk)
+
+# About
+The Embrace Android SDK builds on top of [OpenTelemetry](https://opentelemetry.io) to capture performance data for 
+Android apps, enabling full-stack observability of your system by connecting mobile and backend telemetry in a seamless way.
+
+Telemetry recorded through this SDK can consumed on the Embrace platform for Embrace customers, but it can also be used by those who are
+not Embrace customers to export collected data directly to any OTel Collector. In effect, this SDK is an alternative to [opentelemetry-android](https://github.com/open-telemetry/opentelemetry-android) or using the
```
Comment (davidlawrencer) [inline README.md:21]: to any OTel Collector **or vendor**

File: README.md [hunk 3]
```diff
@@ -8,34 +8,85 @@
...
 
-> :warning: **This is for native Android apps**: Use our Unity, ReactNative and Flutter SDKs for cross-platform apps 
+```kotlin
+internal class MyApplication : Application() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
 
-- [Go to our dashboard](https://dash.embrace.io/signup/) to create an account and get your API key
-- Check our [guide](https://embrace.io/docs/android/integration/) for instructions to integrate the SDK into your app
+        // Exporter setup
```
Comment (davidlawrencer) [inline README.md:67]: // Do your exporter setup before starting the SDK

File: README.md [hunk 4]
```diff
@@ -8,34 +8,85 @@
...
+    }
+}
+```
 
-- Follow our [upgrade guide](https://github.com/embrace-io/embrace-android-sdk/blob/master/UPGRADING.md)
+For details about the features the Embrace Android SDK supports, refer to our [features page](https://embrace.io/docs/android/features/).
 
-# Usage
+## Prospective Embrace Customers
+- If you want to try out Embrace product along with using this SDK, [go to our website](https://dash.embrace.io/signup/) and begin the sign up process.
```
Comment (davidlawrencer) [inline README.md:77]: try out **the** Embrace product

**Review by davidlawrencer — APPROVED** [review]
Looks great! Added some clarifying wording and had one spot where adding a sentence might help, but these are nits.

File: README.md [hunk 5]
```diff
@@ -8,34 +8,85 @@
...
+Telemetry recorded through this SDK can consumed on the Embrace platform for Embrace customers, but it can also be used by those who are
+not Embrace customers to export collected data directly to any OTel Collector. In effect, this SDK is an alternative to [opentelemetry-android](https://github.com/open-telemetry/opentelemetry-android) or using the 
+[OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java) directly for Android apps that want to leverage the 
+OpenTelemetry ecosystem for observability. Currently, only Spans and Logs are supported, but other signals will be added in the future.
 
 # Getting Started
+## Non-Embrace Users
+### Android Project Setup
+
+1. In your app's Gradle file, add a dependency to the latest version of the Embrace Swazzler Gradle plugin: `io.embrace:embrace-swazzler:<version>`.
```
Comment (fractalwrench) [inline README.md:29]: Should contain the same code snippet as the integration docs IMO

File: README.md [hunk 6]
```diff
@@ -8,34 +8,85 @@
...
+   - This plugin is responsible for configuring your app at build time to auto-capture telemetry. This includes:
+     - Updating dependencies to include optional modules that are needed for certain features.
+     - Setting up configuration files to be read at runtime.
+     - Doing bytecode instrumentation to enable the capture of certain telemetry.
+2. In the Gradle files of modules you want to invoke Embrace SDK API methods, add a dependency to the main Embrace SDK module: `'io.embrace:embrace-android-sdk:<version>`.
```
Comment (fractalwrench) [inline README.md:34]: Not strictly required, unless it's a multi module project

File: README.md [hunk 7]
```diff
@@ -8,34 +8,85 @@
...
+3. In the `main` directory of your app's root source folder (i.e. `app/src/main/`), add in a file called `embrace-config.json` that contains `{}` as its only line.
```
Comment (fractalwrench) [inline README.md:35]: We should look at getting rid of this requirement as part of 7.0. Do we have an epic already for that?

File: README.md (hunk 5 above)
Comment (bidetofevil) [reply README.md:29]: Sounds good. I'll try to do that, if not now then later.

File: README.md (hunk 7 above)
Comment (bidetofevil) [reply README.md:35]: Nope but definitely agree. This is very annoying to write lol.

**Review by davidlawrencer — APPROVED** [review]

**Review by fractalwrench — APPROVED** [review]

**Pull request merged**